templates/cli/appwrite.json.twig

```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "https://raw.githubusercontent.com/ScoopInstaller/Scoop/master/schema.json",
+    "version": "{{ sdk.version }}",
+    "description": "Appwrite cli project manager",
+    "license": "BSD-3-Clause",
+    "url": "https://github.com/{{ sdk.gitUserName }}/{{ sdk.gitRepoName | caseDash }}/releases/download/{{ sdk.version }}/{{ language.params.executableName }}-cli-win-x64.exe",
+    "bin": [
+        [
+            "{{ language.params.executableName }}-cli-win-x64.exe",
+            "{{ language.params.executableName|caseLower }}"
+        ]
+    ],
+    "homepage": "https://github.com/{{ sdk.gitUserName|url_encode }}/{{ sdk.gitRepoName|url_encode }}",
+    "hash": "https://github.com/appwrite/sdk-for-cli/releases/download/$version/hashes",

```

Comment:
I don't think this exists. Where's the update to the job that will generate this.

Reply:
You are right. This doesn't exist right now and I have no idea why it's here since I had taken it out to commit the file. My bad that it got pushed but it will be removed until a proper system has been figured out.

Reply:
yes, however, if we want those changes reflected in the JSON file in the commit then the workflow will have to create another commit on the master branch updating the hash. (which personally I don't find very appealing). The two other solutions would be providing the JSON as a release artifact (which I would think is a bit strange) or leaving away the hash for the base file and only having the auto-updater.

The package will work without the hash it just misses the client-side verification but that is only relevant when expecting an MTM attack which is going to be unlikely. Instead, I propose we create the file with the hash embedded and create an automated PR to the main bucket of the scoop installer to update the app there. We don't even have to do this all the time since the auto updater can get the hashes from the URL.